tsconfig.cjs.json

```diff
@@ -0,0 +1,9 @@
+// -*- jsonc -*-
+{
+  "extends": "./tsconfig.json",
+  "include": ["./src/**/*"],
```

Comment:
does this still need to be here even though we have `extends`?

Reply:
no - will remove